Python/README.md

```diff
@@ -1,6 +1,6 @@
 # IBM MQ Python samples
 The python samples are based on https://dsuch.github.io/pymqi/
-and have been tested with python 2.7.10 and 3.5.1
+and have been tested with python 2.7.10, 3.5.1, 3.11.9 and 3.12.5
```

Comment:
Limit the list to those that have tested with all these versions in this bump cycle